Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.4
```

Comment:
Since it's pointing to dev, we don't need to bump the micro. It would be good to put a dev tag on there, since develop should be targeting the next minor release (i.e., 0.2-dev) but we can do that in a separate update if you prefer to keep your PR clean

Reply:
Updated the version to 0.2-dev